apps/emqx/src/emqx_limiter/src/emqx_limiter_schema.erl

```diff
@@ -461,3 +462,12 @@ alias_of_type(_) ->
 
 deprecated_desc(_Field) ->
     <<"Deprecated since v5.0.25">>.
+
+rate_type() ->
+    typerefl:alias("string", rate()).
```

Comment:
This will cause the type to be displayed as "string()" in error messages. Is this the intention?

Reply:
the to_rate input is string. so I don't think it will be confusing.

Reply:
Ok